source/tyr/requirements.txt

```diff
@@ -11,15 +11,14 @@ SQLAlchemy==1.3.3
 Shapely==1.5.0
 Werkzeug==0.15.5
 alembic==1.0.11
-amqp==1.4.9
+amqp==2.1.1
 aniso8601==0.82
-anyjson==0.3.3
 argparse==1.2.1
-billiard==3.3.0.13
-celery==3.1.7
+billiard==3.5.0.5
+celery==4.0.0
```

Comment:
I didn't check all other libs, but it looks weird to not go at least on the last "hotfix" version https://pypi.org/project/celery/#history
So in this case 4.0.2 would probably be more secure (and maybe 4.1.0 to match kirin, or even 4.1.1 to apply the same reasoning).